PowerEditor/src/NppDarkMode.cpp

```diff
@@ -1,4 +1,4 @@
-﻿#include "nppDarkMode.h"
+﻿#include "NppDarkMode.h"
```

Comment:
Finally found out why vs2019 has trouble with finding function definition.

Reply:
So file name is case sensitive now ?

Reply:
@donho
> So file name is case sensitive now ?

It depends on compiler implementation, the most used will usually compile fine, some with warning.

This is more about vs2019 as IDE, as sometimes it can have trouble with connecting the right header to source file,
so I can not use `ctrl` + clicking on function name to go to function declaration or definition.